### PR TITLE
Don't use Base.Callable

### PR DESCRIPTION
**Pull request by nickrobinson251**
it's not exported. i think we can survive without it.

### CODE REVIEW

**Comment by bkamins** [comment]
I am OK with this change.

**Comment by rofinn** [comment]
@bkamins Should this be merged then?

File: src/Missings.jl [hunk 1]
```diff
@@ -216,6 +216,6 @@ julia> passmissing((x,y)->"\$x \$y")(1, 2)
 julia> passmissing((x,y)->"\$x \$y")(missing)
 missing
 """
-passmissing(f::Base.Callable) = PassMissing{typeof(f)}(f)
+passmissing(f) = PassMissing{typeof(f)}(f)
```
Comment (ararslan) [inline src/Missings.jl]: Unrelated to this PR, but I think `typeof(f)` here should actually be `Core.Typeof(f)`.
```julia
julia> typeof(Int)
DataType

julia> Core.Typeof(Int)
Type{Int64}

julia> typeof(sin) === Core.Typeof(sin)
true
```

Comment (nickrobinson251) [reply src/Missings.jl]: will open another PR :)

Comment (nickrobinson251) [reply src/Missings.jl]: (after this one is merged)

Comment (bkamins) [reply src/Missings.jl]: Good point - I would propose to change this in this PR and add appropriate tests for: function, type and functor.

@nickrobinson251 - would you be willing to do this?

Comment (nickrobinson251) [reply src/Missings.jl]: 👍

**Comment by bkamins** [comment]
I propose to wait for @nalimilan to comment on this. After merging all the recent PRs probably we should bump a version of Missings.jl and tag a release.

File: src/Missings.jl (hunk 1 above)
Comment (nickrobinson251) [reply src/Missings.jl]: Let me now if I missed anything 😄

File: test/runtests.jl [hunk 2]
```diff
@@ -139,13 +139,22 @@ using Test, SparseArrays, Missings
...
+    @test passmissing(cuberoot)(27) == 3.0
+    @test isequal(passmissing(cuberoot)(missing), missing)
     @test passmissing(sqrt)(4) == 2.0
     @test isequal(passmissing(sqrt)(missing), missing)
     @test isequal(passmissing(sqrt).([missing, 4]), [missing, 2.0])
     @test passmissing((x,y)->"$x $y")(1, 2) == "1 2"
     @test isequal(passmissing((x,y)->"$x $y")(missing), missing)
     @test_throws ErrorException passmissing(string)(missing, base=2)
 
+    @test passmissing(sin) === Missings.PassMissing{typeof(sin)}(sin)
```
Comment (bkamins) [inline test/runtests.jl]: We also might add tests `passmissing(Int)(1)` and `passmissing(Int)(missing)`

@ararslan Do you think that with `Core.Typeof` the speed of the above will increase?

Comment (nickrobinson251) [reply test/runtests.jl]: added :)

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**